raster/src/main/scala/geotrellis/raster/io/geotiff/CompressedBytesArray.scala

```diff
@@ -0,0 +1,71 @@
+package geotrellis.raster.io.geotiff
+
+import geotrellis.vector.Extent
+import geotrellis.raster._
+import geotrellis.raster.io.geotiff._
+import geotrellis.raster.io.geotiff.reader._
+import geotrellis.raster.io.geotiff.tags._
+import geotrellis.raster.io.geotiff.util._
+
+import java.nio.ByteBuffer
+import monocle.syntax.apply._
+import spire.syntax.cfor._
+
+class CompressedBytesArray(compressedBytes: Array[Array[Byte]]) extends CompressedBytes {
```

Comment:
I agree, those names were just place holders until I could think of something better. I like the names you proposed, and will rename the files and classes with their respective new names.